src/etc/module.xml

```diff
@@ -6,6 +6,7 @@
             <module name="Magento_InventorySales"/>
             <module name="Magento_InventorySalesApi"/>
             <module name="Magento_InventoryShipping"/>
+            <module name="Magento_InventoryInStorePickupSales"/>
```

Comment:
Same here - I'm not sure we should add this. @convenient, thoughts?

Reply:
If its not a hard dependency and the plugin just won't execute, then I think we'll be fine without this.

Reply:
If its not a hard dependency and the plugin just won't execute, then I think we'll be fine without this.